{"name": "第二次进入到九天预报", "status": "passed", "attachments": [{"name": "log", "source": "0103d1b4-7c2c-4a55-986d-39b7e8771b35-attachment.txt", "type": "text/plain"}], "start": 1639634556909, "stop": 1639634567124, "uuid": "5bc449bc-206f-4d8d-83bc-76b63553029d", "historyId": "fecfeadbdf638d2936ba5f8ef809477e", "testCaseId": "b3a253b950ea03bdbf30df8119c94409", "fullName": "testcase.test_nine_day.TestNineDay#test_second_nine_day", "labels": [{"name": "feature", "value": "测试进入九天预报"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_nine_day"}, {"name": "subSuite", "value": "TestNineDay"}, {"name": "host", "value": "OS-201903242231"}, {"name": "thread", "value": "10888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_nine_day"}]}